lua/weapons/weapon_taser.lua

```diff
@@ -106,12 +106,15 @@ function SWEP:Reload() -- remove the attachment
     self:RemoveWire()
 end
 
-function SWEP:Think() -- remove the rope if the player goes too far
+function SWEP:Think() -- remove the rope if the player goes too far and untases people not being tased
     if IsValid(self.AttachedPlayer) then
         if self.Owner:GetPos():DistToSqr(self.AttachedPlayer:GetPos()) > self.MaxHoldDist*self.MaxHoldDist or !self.AttachedPlayer:Alive() then
             self:RemoveWire()
         end
     end
+    if !IsValid(ply.IsTaseredBy) then
```

Comment:
ply is not defined in this context. It looks like UntasePlayer is already called by RemoveWire so I don't think this is the fix

Reply:
Sorry, I'm new to writing lua in garry's mod. I'll try finding a proper fix later